packages/paste-core/components/spinner/__tests__/index.test.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {Theme} from '@twilio-paste/theme';
+import {render, screen} from '@testing-library/react';
+import type {RenderOptions} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {Spinner} from '../src';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+
+expect.extend(matchers);
+
+const TestWrapper = (elements?: Record<string, any>): RenderOptions['wrapper'] => ({children}) => (
+  <Theme.Provider theme="default">
```

Comment:
You don't need the Theme.Provider when immediately wrapping with Customization.Provider